### PR TITLE
[DOC-7406] Publish A-99796

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-7406] Publish A-99796

## Previews
[advisories/a99796.md](https://deploy-preview-16770--cockroachdb-docs.netlify.app/docs/advisories/a99796.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

File: advisories/a99796.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+affected_versions: v22.2.0 to v22.2.7
+advisory_date: 2023-04-17
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+Starting in [CockroachDB v22.2.0](../releases/v22.2.html#v22-2-0), the implementation for `ALTER TABLE..ADD COLUMN` began using the new [declarative schema changer](/docs/{{site.versions["v22.2"]}}/online-schema-changes.html#declarative-schema-changer). However, this implementation introduced a bug when a specific [Data Manipulation Language (DML) schema change operation](https://developer.ibm.com/tutorials/perform-dml-operations-on-cockroachdb/) occurs concurrently with an `UPDATE` or `INSERT` command, and the schema change fails and is rolled back.
```
Comment (lnhsingh) [inline advisories/a99796.md:15]: Link to `alter-table.html#add-column`

Comment (lnhsingh) [inline advisories/a99796.md:15]: Also link to update / insert pages

File: advisories/a99796.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+If a new column family with a single column was created using an `ALTER TABLE..ADD COLUMN` command that was executed concurrently with an `UPDATE` or `INSERT` command, this new column family could be written within any primary index. This only became an issue if the schema change failed and was rolled back. This bug could cause corruption in the primary index, because the primary index would still contain the new data after the added column or column family was cleaned up from the table descriptor. In this situation, the table could become inaccessible.
```
Comment (lnhsingh) [inline advisories/a99796.md:17]: Same links here. Might also want to link "column family"

File: advisories/a99796.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+This is resolved in CockroachDB by PR [#99953](https://github.com/cockroachdb/cockroach/pull/99953), which adds validation to rollback operations to account for this situation.
+
+The fix has been applied to maintenance releases of CockroachDB: [v22.2.8](../releases/v22.2.html#v22-2-8).
+
+This public issue is tracked by [#99796](https://github.com/cockroachdb/cockroach/issues/99796).
+
+## Mitigation
+
+Users of CockroachDB v22.2.0 to v22.2.7 are encouraged to upgrade to [v22.2.88](../releases/v22.2.html#v22-2-8) or a later version.
+
```
Comment (lnhsingh) [inline advisories/a99796.md:32]: Should this be under a "Impact" section?

**Review by lnhsingh — COMMENTED** [review]
Couple nits about adding links and a question about adding a new heading, "Impact"

Once addressed, LGTM (pending tech review)

File: advisories/a99796.md (hunk 3 above)
Comment (mdlinville) [reply advisories/a99796.md:32]: No, I am going off a previous TA and how they are organized: https://www.cockroachlabs.com/docs/advisories/a99049

Comment (mdlinville) [reply advisories/a99796.md:32]: Now I see what you mean. Fixed.

**Review by rimadeodhar — APPROVED** [review]
The technical text LGTM. Thanks Matt!

**Pull request merged**